readyapi4j-swagger/src/test/java/com/smartbear/readyapi4j/swagger/SwaggerTestStepBuilderTest.java

```diff
@@ -29,8 +36,37 @@ public void testExistingOperation() throws Exception {
     public void testMissingOperation() throws Exception {
         try {
             petstore.operation("tjoho");
-            assertFalse("Expected IllegalArgumentException for invalid operationId", true);
+            fail("Expected IllegalArgumentException for invalid operationId");
```

Comment:
I missed this when reviewing Ole's code so I complain now instead ...

The cleanest and most readable way to verify this is to use the expected attribute in the @Test annotation:

@Test(expected = IllegalArgumentException.class)